packages/cli/src/rpc/navie/metadata.ts

```diff
@@ -0,0 +1,52 @@
+import { NavieRpc } from '@appland/rpc';
+import { RpcHandler } from '../rpc';
+
+export function navieMetadataV1(): RpcHandler<
+  NavieRpc.V1.Metadata.Params,
+  NavieRpc.V1.Metadata.Response
+> {
+  return {
+    name: NavieRpc.V1.Metadata.Method,
+    handler: () => ({
+      welcomeMessage:
+        'Navie can help you answer questions about your codebase, plan solutions, and generate code. Enter `@` to see a list of commands.',
```

Comment:
Do we want to change these to `/` commands at this point?

Reply:
If possible, I don't want to tackle that as part of this change. All the pieces are in place so that we can update the welcome message and input placeholder returned by the `v1.navie.metadata` RPC method as a follow up.